rholang/reference_doc/data_structures/README.md

```diff
@@ -1,13 +1,47 @@
 ## Data Structures
 
-### Structures
+All computation can be encoded in the raw rho calculus just as it can be in the lambda calculus. Convincing your self of this is an interesting and rewarding endeavor for the mathematically inclined. Rholang strives to b a useful general purpose programming language, and therefore introduces some additional native data structures.
 
-list, tuple, set, map
+All data structures are processes and [can be quote] todo insert link, to create channels.
 
+All data structures are immutable, and their methods return new data structures with the requested change (the original is not modified).
+
+
+### Tuple
+A tuple is an ordered collection of 1 or more processes. The constituent processes need not be of the same type.
+
+#### Basic syntax
+`(1, 2, "hello", *someChan)`
+
+#### `tuple.nth`
```

Comment:
I like including the method here and in the methods doc. Makes info easier to find/remember.